catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/impl/ResourceCacheImplTest.java

```diff
@@ -169,8 +169,8 @@ public void testPutThenGet() {
      * in the pending cache list.
      */
     @Test
-    public void testPutThenGetNotPending() {
-        MetacardImpl metacard = new MetacardImpl();
+    public void testPutThenGetNotPending() throws URISyntaxException {
+        MetacardImpl metacard = generateMetacard();
```

Comment:
@shaundmorris i can change them if you want, but they were already there when i fixed the failing tests